packages/orbit-design-tokens/src/dictionary/formatters/jsTokens.ts

```diff
@@ -117,7 +117,17 @@ const typescriptFactory = (allProperties: Dictionary["allProperties"], complete
 
       return createObjectProperty(name, `boxShadow(${boxShadowValue})`);
     }
-    return createObjectProperty(name, plainValue);
+
+    /**
+     * This ensures tokens like 'foundations.borderRadius.50'
+     * will be correctly written as 'foundations.borderRadius["50"]'
+     * and be correct JavaScript code.
+     */
+    const val =
```

Comment:
This probably could be moved to the `getValue` helper, but I'm not sure of the possible consequences of putting it there.
Probably something to inquire.